src/ResourceManager/Compute/Commands.Compute/Common/ComputeClientBaseCmdlet.cs

```diff
@@ -50,7 +50,6 @@ public ComputeClient ComputeClient
         public override void ExecuteCmdlet()
         {
             base.ExecuteCmdlet();
-            ComputeAutoMapperProfile.Initialize();
```

Comment:
In the past, this function is called from every cmdlet run, so it can make sure all mapping are initialized.

How does this work now? How can it make sure all users' cmdlets will have the mapping initialized beforehand?

Do we have a live test for it?

Reply:
@huangpf the `ComputeAutoMapperProfile.Mapper` is lazily instantiated now, so whenever `ComputeAutoMapperProfile.Mapper.Map` is called, it will check if the `Mapper` object has been initialized, and if not, it will call the `Initialize()` function to create the mappings.

Before Hovsep's changes, this was how the initialization was essentially done (check if the `Mapper` had been initialized, and if not, initialize it), except it was being done in the `ExecuteCmdlet()` function rather than when the `Mapper` was being used.

Reply:
@cormacpayne
Our main concern is that the live tests should be run and pass. Other than that it looks fine to me. Thanks.